deployment/allyabase_setup.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+# Dependencies:
+# git
+# nodejs
+# npm
+
+set -e
+
+tmpDir="${TMPDIR:-${XDG_RUNTIME_DIR:-/tmp}}"
+buildDir="${1:-$tmpDir}"
+[[ $buildDir != /* ]]&& buildDir="${PWD%/}/$buildDir"
+buildDir="${buildDir#./}/allyabase"
+ecosystem_config='ecosystem.config.js'
+services=(
+    'julia'
+    'continuebee'
+    'joan'
+    'pref'
+    'bdo'
+    'fount'
+    'addie'
+    'aretha'
+    'sanora-dot-club'
```

Comment:
I always do these in alphabetical order otherwise I can't tell if I've missed any. I'll rearrange after I merge, but thought I'd mention it since it helps with random strings in lists in code.